internal/ethapi/api.go

```diff
@@ -888,6 +888,40 @@ func (diff *StateOverride) Apply(state *state.StateDB) error {
 	return nil
 }
 
+// BlockOverrides is a set of header fields to override.
```

Comment:
> Any reason why you defined this in internal/ethapi instead of eth/tracers?

I didn't really think about it, I guess it wound up there because that's where `StateOverride` is.

Reply:
Ok I guess we can keep it there in case we want to add `BlockOverride` to `eth_call` as well

Reply:
OMG, sorry, I completely messed up your conversation, I was commenting on other part. Completely disregard everything I wrote above.